assets/js/components/Notice/CTAButton.js

```diff
@@ -49,6 +58,8 @@ CTAButton.propTypes = {
 	label: PropTypes.string.isRequired,
 	disabled: PropTypes.bool,
 	inProgress: PropTypes.bool,
-	onClick: PropTypes.func.isRequired,
```

Comment:
Removed because in some instances, a URL us passed as opposed to an onClick callback.

Reply:
It might be a good idea to use `onClick` in all cases instead (could be follow up).

Reply:
Thanks @aaemnnosttv going to flag this as a follow up.